### PR TITLE
Feature of reporting implemented. (by mail)

### PR DESCRIPTION
**Pull request by Netmisa**
#### What I did

I implement reporting command

#### How I did it

I get location patches of the last 7 days

#### How to verify it
```
Usage:
  hermod:report:send <to> [<cc>]...

Arguments:
  to                    Email of recipient in to
  cc                    Emails of recipients in cc
```

```
$ ./bin/console hermod:report:send remy.abikhalil@canaltp.fr
[OK] [2017-09-27 01:31:58] - Message sended to: "remy.abikhalil@canaltp.fr" cc: ""
```

### CODE REVIEW

File: src/AppBundle/Resources/views/Emails/registration.html.twig [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Bonjour,<br>
+<br>
+Vous trouverez en pièce jointe la liste des signalements d'arrêt sur les 7 derniers jours.<br>
```
Comment (dvdn) [inline src/AppBundle/Resources/views/Emails/registration.html.twig]: Suggestions : 
Trouvez ci-joint les signalements sur arrêts de ces 7 derniers jours.
Merci, Bien cordialement, > useless
switch 2 last lines

File: src/AppBundle/Services/LocationPatch.php [hunk 2]
```diff
@@ -58,4 +58,30 @@ public function create(User $user, $data, bool $withReporterLocation = false)
         $this->createLocation($user, $data, $withReporterLocation);
         $this->em->flush();
     }
+
+    public function getCsvReportByPeriod()
```
Comment (Fahmus) [inline src/AppBundle/Services/LocationPatch.php]: it's better to declare a parameter in parameters.yml
public function getCsvReportByPeriod($days)

Comment (Netmisa) [reply src/AppBundle/Services/LocationPatch.php]: It's an error I call it from [here](https://github.com/CanalTP/Hermod/pull/50/files#diff-b3e26ca070975c6d3c1fd19d6b692413R25), I should put it in argument.
Thanks

File: src/AppBundle/Resources/views/Emails/registration.html.twig (hunk 1 above)
Comment (Netmisa) [reply src/AppBundle/Resources/views/Emails/registration.html.twig]: Done

File: src/AppBundle/Services/LocationPatch.php (hunk 2 above)
Comment (Fahmus) [reply src/AppBundle/Services/LocationPatch.php]: You cannot also put it in parameters.ini?
So if you would change periodicity a day you change directly the parameter :)

File: src/AppBundle/Command/SendReportCommand.php [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+        ;
+    }
+
+    private function getMessage($to, $cc) : \Swift_Message
+    {
+        $templating = $this->getContainer()->get('templating');
+
+        return (new \Swift_Message())
+            ->setSubject('[HERMOD] - Export des signalements d\'arrêt')
+            ->setFrom(['hermod@kisio.org' => 'Hermod team'])
```
Comment (Fahmus) [inline src/AppBundle/Command/SendReportCommand.php]: the same thing in parameters.ini:
mail.subject
mail.from 
...

Comment (Netmisa) [reply src/AppBundle/Command/SendReportCommand.php]: Thanks , but this informations will not change. Actually we have only `$to` and `$cc` in arguments of the command.
I think this command will not be the right solution and should be removed in the future, next step we will do board.

Thanks for your reviews

**Pull request merged**